controller/controller_server/csi_controller_server.py

```diff
@@ -84,7 +84,7 @@ def CreateVolume(self, request, context):  # pylint: disable=too-many-branches
                 logger.debug(array_mediator)
                 # TODO: CSI-1358 - remove try/except
                 try:
-                    volume_full_name, volume_prefix = self._get_volume_name_and_prefix(request, array_mediator)
+                    volume_full_name = self._get_volume_name(request, array_mediator)
```

Comment:
consider also removing `full_` here (since there is no more prefix var to distinguish from)

Reply:
there is a prefix var to distinguish from, its not part of the volume name from the request, that why it called full name, not because we used to get the prefix too..

Reply:
it [used to be](https://github.com/IBM/ibm-block-csi-driver/blame/a189d1a1043f157051461176a2710a3c73252be4/controller/controller_server/csi_controller_server.py#L69) without `full_`, and [was added](https://github.com/IBM/ibm-block-csi-driver/blame/618a82a8bb6f20db5dc43bc515cbb7bd4e30f1fa/controller/controller_server/csi_controller_server.py#L60) during the ds8k hash addition.
but if you think it should still be distinguished from the original `request.name`, then I'm not sure `full_` is still accurate, since it might also be hashed.
`final`/`in_storage` may be more accurate (same for the "get" functions)

Reply:
done